### PR TITLE
Created methods in Lobby Class and minor changes

### PR DESCRIPTION
**Pull request by Ji-Justin**
Fleshed out methods in Lobby Class, moved room variables to Game Class, removed application.properties from .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,4 +31,3 @@ build/
 
 ### VS Code ###
 .vscode/
-src/main/resources/application.properties
```
Comment (RileyChampion) [inline .gitignore:34]: Why did you remove this from the gitignore?

File: src/main/java/com/example/TwoRoomsBoom/models/Lobby.java [hunk 2]
```diff
@@ -17,9 +21,85 @@
...
+
+    public void kickPlayer(Player player) {
+        if (this.players.contains(player)) {
+            this.players.remove(player);
+        }
+    }
+
+    public void addRole(Role role) {
+        // Can have multiple Blue/Red team role in play but other roles cannot
+        if (role.getName().equals("Blue Team") || role.getName().equals("Red Team")) {
```
[the remark this answers is not in the document]
Comment (RileyChampion) [reply src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:50]: Maybe a different data structure? just ideas yk

File: src/main/java/com/example/TwoRoomsBoom/models/Lobby.java [hunk 3]
```diff
@@ -17,9 +21,85 @@
...
+            this.rolesAdded.add(role);
+        }
+    }
+
+    public void removeRole(Role role) {
+        if (this.rolesAdded.size() > 0 && this.rolesAdded.contains(role)) {
+            // Removes both Blue/Red Team
+            if (role.getName().equals("Blue Team") || role.getName().equals("Red Team")) {
+                Iterator<Role> iter = rolesAdded.iterator();
+                while (iter.hasNext()) {
```
Comment (RileyChampion) [inline src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:68]: Yea I think if we were to add additional roles this would be a long operation especially if there's a lot of adding and removing of roles. lowkey think that the roles should be stored differently

**Review by RileyChampion — COMMENTED** [review]
Looks good overall! But when I was looking at the way that "Red Team" and "Blue Team" roles were being added it lowkey felt like it was a lot of unnecessary work to add and remove duplicates of those roles. Thinking maybe a different data structure would be better to store the roles like a hashmap?

File: .gitignore (hunk 1 above)
Comment (Ji-Justin) [reply .gitignore:34]: Originally it was added to store and use an api key but not anymore

File: src/main/java/com/example/TwoRoomsBoom/models/Lobby.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ji-Justin) [reply src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:50]: I think only the Blue/Red team role is the only exception but I do know that in the physical game there's up to 7 team cards each

[the remark this answers is not in the document]
Comment (Ji-Justin) [reply src/main/java/com/example/TwoRoomsBoom/models/Lobby.java:50]: But yeah I do agree I didn't like having to do this approach when I implemented it, a hashmap might be a good idea

**Review by RileyChampion — APPROVED** [review]
lgtm
